### DISCUSSION TITLE
Blockchain, smart contract, solidity

### DISCUSSION THREAD

**Discussion by Ugochi-Eze** [smartcontractkit/full-blockchain-solidity-course-py#1511] [General]
I appreciate the work done to put together the training on blockchain, smart contract, solidity on python edition and JavaScript edition. Although I recently started the training on python edition, I believe I will have learnt a lot by the time I am done. Thank you so much for all effort and knowledge.

**Comment by cromewar** [comment]
@PatrickAlphaC 
Patrick put a lot of efforts on this course, I'm glad you are enjoying it ;).

**Reply by PatrickAlphaC** [reply]
And Cromewar has been a superstar helping everyone continue to learn!

**Comment by Ugochi-Eze** [comment]
I will, thank you.

**Comment by Ugochi-Eze** [comment]
Hi,
Do you mean Vasiliy
